pints/_mcmc/_adaptive_covariance.py

```diff
@@ -168,25 +174,42 @@ def tell(self, fx):
         # Adapt covariance matrix
         if self._adaptive:
             # Set gamma based on number of adaptive iterations
-            gamma = self._adaptations ** -0.6
+            self._gamma = self._adaptations ** -self._eta
             self._adaptations += 1
 
-            # Update mu, log acceptance rate, and covariance matrix
-            self._mu = (1 - gamma) * self._mu + gamma * self._current
-            self._loga += gamma * (accepted - self._target_acceptance)
-            dsigm = np.reshape(self._current - self._mu, (self._dimension, 1))
-            self._sigma = (
-                (1 - gamma) * self._sigma + gamma * np.dot(dsigm, dsigm.T))
+            # For localised AM
+            if self._localised:
+                self._fit_gaussian_mixture()
```

Comment:
This method should be defined in this base class. The base class shouldn't know anything about it's children!

Reply:
Perhaps should have a name like `update_...` to match with the others? In non-local mode we update mu and sigma, but in local mode we fit a gaussian mixture?